--- rpc2-2.10/configure.in~	2010-03-22 13:18:33.000000000 -0600
+++ rpc2-2.10/configure.in	2010-07-17 11:19:49.000000000 -0600
@@ -15,7 +15,7 @@
 dnl   first to 0
 dnl - if any interfaces were added, increment third
 dnl - if any interfaces were removed, set third to 0
-CODA_LIBRARY_VERSION(5, 9, 4)
+CODA_LIBRARY_VERSION()
 
 CONFIG_DATE=`date +"%a, %d %b %Y %T %z"`
 AC_SUBST(CONFIG_DATE, "$CONFIG_DATE", [Date when configure was last run])
@@ -64,7 +64,7 @@
      #include <netinet/in.h>])
 
 dnl Checks for compiler characteristics.
-CODA_CC_FEATURE_TEST(Wall)
+CODA_CC_FEATURE_TEST()
 
 dnl Checks for library functions.
 AC_CHECK_FUNCS(ffs iopen getaddrinfo gai_strerror getipnodebyname)
